public/PlayHands.js

```diff
@@ -0,0 +1,230 @@
+"use strict";
+import { sleepNow } from "./Helpers.js";
+
+export default class PlayHands {
+  constructor(board) {
+    this.board = board;
+    this.controlsContainerEle = this.board.boardContainerElem;
+  }
+
+  rendnerCotnrols() {
```

Comment:
There is a typo here:
```suggestion
  renderControls() {
```